docs/backup.md

```diff
@@ -0,0 +1,42 @@
+# Backup/Restore overview
```

Comment:
we need to "hang this up" somewhere at the docs tree in `mkdocs.yaml` file, otherwise it's not visible by default at all.